Makefile

```diff
@@ -53,11 +53,11 @@ build-terraform:
 	@if [ -z "$(RESOURCE)" ]; then \
 		echo "==> Generating source code for terraform from resource list..."; \
 		cd $(MMROOT) && \
-		jq '.[]' $(ROOT)/resources.json | xargs -I '{}' bundle exec compiler -d -p $(SPECROOT)/'{}' -e terraform -o $(TFROOT)/$(TFREPO)/; \
+		jq '.[]' $(ROOT)/resources.json | xargs -I '{}' bundle exec compiler -d -p $(SPECROOT)/'{}' -e terraform -c azure -o $(TFROOT)/$(TFREPO)/; \
```

Comment:
Is `azure` switch newly introduced in `magic-modules` ? You've have to run `git submodule update --remote` to sync to latest commit, commit and push it.

Reply:
I noticed there is the _git submodule update_  already in [Makefile](https://github.com/Azure/magic-module-specs/blob/master/Makefile#L38). 

Is that means I need to update the submodule in my local repository?